arwen/interface.go

```diff
@@ -145,6 +145,13 @@ type RuntimeContext interface {
 	BigIntAPIErrorShouldFailExecution() bool
 	ManagedBufferAPIErrorShouldFailExecution() bool
 	ExecuteAsyncCall(address []byte, data []byte, value []byte) error
+	EnableGasTrace()
+	DisableGasTrace()
+	TraceGasUsed(functionName string, initialGasLeft uint64)
```

Comment:
we could have the Gas tracing as a separate component - outside of runtime context. It would be better.